draft-nuttycom-funding-allocation.rst

```diff
@@ -0,0 +1,609 @@
+   involvement in funding decisions.
+
+4. **Community Involvement**: The current system provides minimal formal input
+   from the community regarding what projects should be funded, leading to a
+   misalignment between funded projects and community priorities.
+
+5. **Moving Towards a Non-Direct Funding Model**: There is strong community
+   support for a non-direct Dev Fund funding model. Allocating funds to a
+   Deferred Dev Fund Lockbox incentivizes the development of a decentralized
+   mechanism for the disbursement of the locked funds.
+
+By addressing these issues, this proposal aims to ensure sustainable,
+efficient, and decentralized funding for essential activities within the Zcash
+ecosystem.
+
+Requirements
+============
+
+1. **In-Protocol Lockbox**: The alternatives presented in this ZIP depend upon
+   the Lockbox Funding Streams proposal [#zip-lockbox-funding-streams]_.
+
+2. **Regulatory Considerations**: The allocation of funds should minimize
+   regulatory risks by avoiding direct funding of specific organizations. The
+   design should ensure compliance with applicable laws and regulations to
+   support the long-term sustainability of the funding model.
+
+Non-requirements
+================
+
+The following consideratiosn are explicitly deferred to future ZIPs and are not
+covered by this proposal:
+
+1. **Disbursement Mechanism**: The exact method for disbursing the accumulated
+   funds from the lockbox is not defined in this ZIP. The design,
+   implementation, and governance of the disbursement mechanism will be
+   addressed in a future ZIP. This includes specifics on how funds will be
+   allocated, the voting or decision-making process, and the structure of the
+   decentralized mechanism (such as a DAO).
+
+2. **Regulatory Compliance Details**: The proposal outlines the potential to
+   reduce regulatory risks by avoiding direct funding of US-based
+   organizations, but it does not detail specific regulatory compliance
+   strategies. Future ZIPs will need to address how the disbursement mechanism
+   complies with applicable laws and regulations.
+
+3. **Impact Assessment**: The long-term impact of reallocating a portion of the
+   block subsidy to the lockbox on the Zcash ecosystem, including its effect on
+   miners, developers, and the broader community, is not analyzed in this ZIP.
+   Subsequent proposals will need to evaluate the outcomes and make necessary
+   adjustments based on real-world feedback and data.
+
+Specification
+=============
+
+The following alternatives all depend upon the Lockbox Funding Streams proposal
+[#zip-lockbox-funding-streams]_ for storage of funds into a deferred value
+pool.
+
+Some of the alternatives described below do not specify a termination height
+for the funding streams they propose. In these cases, the termination height
+is set to `u32::MAX_VALUE`. A future network upgrade is required in order for
+these streams to be terminated.
```

Comment:
This last sentence is superfluous; `draft-nuttycom-lockbox-streams` defines how ZIP 207 is amended to support the lockbox, and ZIP 207 already defines an "active funding stream":
> An active funding stream at a given block height is defined as a funding stream for which the block height is less than its end height, but not less than its start height.

as well as the process for modifying funding streams:
> Funding stream definitions can be added, changed, or deleted in ZIPs associated with subsequent network upgrades, subject to the ZIP process.

It would be equally valid, and equally superfluous, to say "A future network upgrade is required in order for the streams that do not terminate at `u32::MAX_VALUE` to extend beyond their current termination height."

What _would_ be a clarifying improvement here is to encode the intention that `u32::MAX_VALUE` is being used to mean "no end height":
```suggestion
is set to `u32::MAX_VALUE`. A future network upgrade that alters the
maximum possible block height MUST also alter these termination heights.
```

Reply:
My rationale for calling out that it could be modified by a future ZIP is to clarify it for voters who are not familiar with the ZIP process, and who might mistakenly think that such rules cannot be changed in the future (which is a common error I have observed.)

Reply:
If that is the motivation for calling it out, then it should be stated neutrally: point out that end heights for any stream can be added, modified, or removed by future ZIPs.